easy-install.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+import argparse
+import subprocess
+import os
+import sys
+import time
+from shutil import which
+import platform
+from secrets import token_bytes
+from base64 import b64encode
+
+CRED = "\033[31m"
+CEND = "\033[0m"
+CGRN = "\33[92m"
+CYLW = "\33[93m"
+
+
+def clone_frappe_docker_repo() -> None:
+    try:
+        subprocess.run(
+            ["git", "clone", "https://github.com/frappe/frappe_docker"], check=True
+        )
+    except Exception as e:
+        print(f"\n{CRED}Cloning frappe_docker Failed{CEND}\n\n", e)
```

Comment:
Log entire exception somewhere so we can ask for more info when required. Printing exception only prints final message. 


This applies everywhere in the script where exceptions are caught and generic message is printed. We'd need some input from those failures to fix them :)

Reply:
use `loggin` module with `exc_info=True`

Log as much as you can so we get decent input for fixing issues when they inevitably come.